test/CMakeLists.txt

```diff
@@ -45,6 +45,10 @@ foreach(name ${listOfTestNames})
   # Make sure the test passes
   set_tests_properties(Test_${name} PROPERTIES PASS_REGULAR_EXPRESSION "ALL PASSED")
 
+  # Set TSAN_OPTIONS for the test
+  set_tests_properties(Test_${name} PROPERTIES
```

Comment:
Does this have an effect on any other builds? (where we don't need/want sanitize)

Reply:
I don't think so, since TSAN is for Thread SANitizer, and it's an environment variable for running the tests, not a build flag but I put it in an if for `NATS_SANITIZE` only just in case.